src/material/form-field/form-field-control.ts

```diff
@@ -63,6 +63,12 @@ export abstract class MatFormFieldControl<T> {
    */
   readonly autofilled?: boolean;
 
+  /**
+   * Value of `aria-describedby` that should be merged with the described-by ids
+   * which are set by the form-field.
+   */
+  readonly userAriaDescribedBy?: string;
```

Comment:
can we add a bit about this to the docs?

Reply:
yeah, that's fair. Added it to the guide and updated our custom control example.